planning/scenario_selector/src/scenario_selector_node/scenario_selector_node.cpp

```diff
@@ -88,19 +88,35 @@ bool isInLane(
   const std::shared_ptr<lanelet::LaneletMap> & lanelet_map_ptr,
   const geometry_msgs::msg::Point & current_pos)
 {
-  const auto & p = current_pos;
-  const lanelet::Point3d search_point(lanelet::InvalId, p.x, p.y, p.z);
-
-  std::vector<std::pair<double, lanelet::Lanelet>> nearest_lanelets =
-    lanelet::geometry::findNearest(lanelet_map_ptr->laneletLayer, search_point.basicPoint2d(), 1);
-
-  if (nearest_lanelets.empty()) {
-    return false;
-  }
-
-  const auto nearest_lanelet = nearest_lanelets.front().second;
-
-  return lanelet::geometry::within(search_point, nearest_lanelet.polygon3d());
+  using lanelet::BasicPoint2d;
+  using lanelet::BoundingBox2d;
+
+  const lanelet::Point2d search_point(lanelet::InvalId, current_pos.x, current_pos.y);
+
+  const double search_width = 20;  // [m]
+  const auto search_box = BoundingBox2d(
+    BasicPoint2d(current_pos.x - search_width, current_pos.y - search_width),
+    BasicPoint2d(current_pos.x + search_width, current_pos.y + search_width));
```

Comment:
[nits] Since these are not referenced in the lambda function, I think it's better to move them right before `const auto resutl = lanelet_map_ptr->laneletLayer.searchUntil(search_box, load_contains_search_point);` to minimize the scope.

Reply:
Thanks. fixed in https://github.com/autowarefoundation/autoware.universe/pull/824/commits/800b55b21bd3d42ce80cb324676c91b904279a5f